Common/Wrapper/TraktManager.swift

```diff
@@ -112,12 +115,15 @@ public class TraktManager {
     
     // MARK: - Setup
     
-    public func setClientID(clientID: String, clientSecret secret: String, redirectURI: String) {
+    public func setClientID(staging: Bool, clientID: String, clientSecret secret: String, redirectURI: String) {
```

Comment:
I would either move `staging` to the end of the parameter list, or change the signature to make more sense. Maybe `setup` or `setupTrakt`. Also it would be good to set the default value of staging to `false`.